components/sup/src/config.rs

```diff
@@ -27,8 +27,6 @@ use std::mem;
 use hcore::package::PackageIdent;
 
 use error::{Error, SupError};
-use gossip::server::GOSSIP_DEFAULT_PORT;
-use topology::Topology;
```

Comment:
Was the topology import removal here intentional? (Seems to be the source of the build failure in travis).